src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -409,6 +425,15 @@ def frequency() -> GridFrequency:
     return _get().frequency()
 
 
+def voltage() -> VoltageStreaming:
+    """Return the 3-phase voltage.
```

Comment:
Super minor: maybe mention here too "for the microgrid".

Reply:
done